src/swap/SwapReviewScreen.tsx

```diff
@@ -111,9 +111,9 @@ export function SwapReviewScreen({ route }: Props) {
       return new BigNumber(0)
     }
 
-    return getMaxGasFee(quote.preparedTransactions.transactions)
-      .shiftedBy(-feeCurrencyToken.decimals)
-      .times(feeCurrencyToken.priceUsd)
```

Comment:
we are showing the network fee in token units on the review screen, so this is incorrect...

Reply:
Damn, good catch, I need to fix this then for today's release 😄

Reply:
if we merge this then we don't need to fix it separately? :D